hibernate-reactive-core/src/test/java/org/hibernate/reactive/BaseReactiveTest.java

```diff
@@ -141,9 +141,14 @@ public void before(TestContext context) {
 
 		final Async async = context.async();
 		Handler<AsyncResult<org.hibernate.SessionFactory>> result = r -> {
```

Comment:
Must say I'm not at all a fan of the assigning-an–anonymous-function-to-a-local-variable idiom. I would inline this local. But that's orthogonal to this patch.